### DISCUSSION TITLE
Any plans to release DJL extension library on maven

### DISCUSSION THREAD

**Discussion by victorex27** [deepjavalibrary/djl#1724] [Development]
Hi All.
I searched through the maven repo and could not find the extension library. Are there plans to release it there? What is my next best option if that is not possible?

I am trying to use the Voice to text part in a maven-run Java Fx project.

**Comment by frankfliu** [comment]
@victorex27 
We do release djl extension regularly.

I think you are referring to [audio](https://github.com/deepjavalibrary/djl/tree/master/extensions/audio) module. This is a new extension, and will be release in 0.18.0 release. We are planning to release 0.18.0 by late Jun and early Jul.

For the time being you can use SNAPSHOT release:

```
    <repositories>
        <repository>
            <id>djl.ai</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
        </repository>
    </repositories>

    <dependencies>
        <dependency>
            <groupId>ai.djl.audio</groupId>
            <artifactId>audio</artifactId>
            <version>0.18.0-SNAPSHOT</version>
        </dependency>
    </dependencies>

```

**Reply by victorex27** [reply]
@frankfliu many thanks.
![Screenshot from 2022-06-20 08-05-38](https://user-images.githubusercontent.com/14342098/174544429-dac79f28-84a4-4573-a08e-27aca4f522ff.png)

It says unresolved dependency: 'ai.djl.audio:jar:0.18.0-SMNAPSHOT'.
I dropped a screenshot of the error.

**Comment by frankfliu** [comment]
The SNAPSHOT jar is published to sonatype: https://oss.sonatype.org/#nexus-search;quick~ai.djl.audio

The package name was wrong, it should be: `ai.dj.audio`

```
<dependency>
  <groupId>ai.djl.audio</groupId>
  <artifactId>audio</artifactId>
  <version>0.18.0-SNAPSHOT</version>
</dependency>
```
